Dockerfile

```diff
@@ -1,21 +1,21 @@
 FROM golang:1.17.6 as builder
 
-MAINTAINER ElrondNetwork
+MAINTAINER MultiversX
 
-WORKDIR /elrond
+WORKDIR /mx
```

Comment:
should we keep this as mx or multiversx ?

Reply:
renamed to multiversx to be more clear